HACKING.md

```diff
@@ -25,34 +25,35 @@ On Fedora:
 On older OS releases you can use the [n utility](https://github.com/tj/n) to
 get a current version of nodejs.
 
-When relying on CI to run the test suite, this is all that is
-necessary to work on the JavaScript components of Cockpit.
+When relying on CI to run the test suite, this is all that is necessary to work
+on the JavaScript components of Cockpit.
 
-To actually build the Cockpit binaries themselves from source
-(including to run the integration tests locally), you will need
-additional header files and other components. Check
-`tools/cockpit.spec` for the concrete Fedora build dependencies.
+To build the Cockpit binaries from source, required to run the integration tests
+locally (see [testing README](test/README.md)), you will need additional header
+files and other components. The following should work in a fresh Git clone:
 
-Note that `tools/cockpit.spec` is a template filled in by
-`tools/gen-spec-dependencies`, and cannot be directly parsed by dnf.
-The following should work in a fresh Git clone:
-
-    $ sudo dnf install dnf-utils
+    $ sudo dnf install dnf-utils python-srpm-macros
     $ TEMPFILE=$(mktemp -u --suffix=.spec) && \
       sed 's/%{npm-version:.*}/0/' tools/cockpit.spec >$TEMPFILE && \
       sudo dnf builddep --spec $TEMPFILE && \
       rm $TEMPFILE
 
+Note that `tools/cockpit.spec` is a template filled in by
+`tools/gen-spec-dependencies` when building RPMs, and cannot be directly parsed
+by dnf.
+
 For running the browser unit tests, the following dependencies are required:
 
-    $ sudo dnf chromium-headless dbus-daemon
+    $ sudo dnf install chromium-headless dbus-daemon
 
 For running integration tests, the following dependencies are required:
 
     $ sudo dnf install curl expect xz rpm-build chromium-headless \
-        libvirt-daemon-kvm libvirt-client python3-libvirt
+        libvirt-daemon-kvm libvirt-client python3-libvirt python3-pyflakes \
+        python3-pycodestyle python3-pyyaml
```

Comment:
pyflakes and pycodestyle are not necessary for the integration tests. They are just being used by `make check`, and even that just skips these two if the packages are not installed. Doesn't do much harm, though.